src/vaadin-crud-grid.html

```diff
@@ -57,6 +61,21 @@
           }
         }
 
+        __onHideEditColumnChange(hideEditColumn) {
+          if (!this.innerHTML == '') {
```

Comment:
Nit: not sure if it's bulletproof. Can we use smth like `!this.firstChild`? 🤔

Reply:
I used the same check that is being used a bit down on the code, but changed to use the suggested one.